source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -225,4 +230,20 @@ execute a find operation on a collection of ``RecordTree`` types:
    .. output::
       :language: sh
 
-      RecordTree[content=Ranginui, left=RecordTree[content=..., left=RecordTree[content=..., right=RecordTree[content=Ikatere...]]
\ No newline at end of file
+      RecordTree[content=Ranginui, left=RecordTree[content=..., left=RecordTree[content=..., right=RecordTree[content=Ikatere...]]
+
+.. _fundamentals-records-get-codec:
+
+Retrieve the Record Codec
+-------------------------
```

Comment:
Comment: I wonder if this whole section should be moved higher on the page.

Reply:
I don't think that is necessary. You only need to get the codec itself if you are creating a custom codec, which the rest of the topic does not get into. However, answering this comment, I realize that should be specified. I will add it to the text.